### PR TITLE
Avoiding crashes for unexpected filenames

### PR DESCRIPTION
**Pull request by brenoaraujolima**
Atificially depends on #54 

Avoiding app crashes 'cause of filenames with special chars. 

Added script to examplify this case

### CODE REVIEW

File: client_example/upload_reqs/filename_unparseable [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+export LC_ALL=C.UTF-8 #prevents grep error
```
Comment (uxjp) [inline client_example/upload_reqs/filename_unparseable:2]: this line is not necessary if you aren't using `grep`

File: client_example/upload_reqs/all_file_req [hunk 2]
```diff
@@ -5,6 +5,6 @@ script_path="./req"
 
 for file in $folder_path/*; do
     if [ -f "$file" ]; then
-        bash $script_path $file
+        bash $script_path "$file"
```
[the remark this answers is not in the document]
Comment (brenoaraujolima) [reply client_example/upload_reqs/all_file_req:8]: Done.

File: .github/workflows/valid-image-generation-test.yml [hunk 3]
```diff
@@ -47,9 +48,10 @@ jobs:
...
       - name: Run Docker container
         run: |
-          (timeout 30s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
+          (timeout 60s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
           if [ $? -eq 124 ]; then
             echo "OK, timeout error is acceptable."
           else
             exit $?
           fi
+        continue-on-error: true
```
Comment (jeffque) [inline .github/workflows/valid-image-generation-test.yml:57]: Does not smells fine

Comment (brenoaraujolima) [reply .github/workflows/valid-image-generation-test.yml:57]: It's inherit from previous PR https://github.com/GeoSales-Evolution/multi-tenant-uploader/pull/54

File: .github/workflows/valid-image-generation-test.yml [hunk 4]
```diff
@@ -47,9 +48,10 @@ jobs:
 
       - name: Run Docker container
         run: |
-          (timeout 30s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
+          (timeout 60s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
```
[the remark this answers is not in the document]
Comment (uxjp) [reply .github/workflows/valid-image-generation-test.yml:51]: 30s wasn't enough to detect error to connect to the mongo database.

File: .github/workflows/valid-image-generation-test.yml (hunk 3 above)
Comment (brenoaraujolima) [reply .github/workflows/valid-image-generation-test.yml:57]: Already mapped in https://github.com/GeoSales-Evolution/multi-tenant-uploader/issues/57

**Pull request merged**